docs/docs/general/03-Guides/reporter.md

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_position: 8
+---
+
+# AskUI Reporter
+We provide a package `askui-reporters` with reporters ready-to-use to create reports for your AskUI runs.
+
+On this page you will learn how to use the Allure-reporter from this package and how you can implement your own reporter for your reporting tool.
+
+Also [check out the repository](https://github.com/askui/askui-reporters) for more details.
+
+## Installation
+Install `@askui/askui-reporters` as a dev-dependency:
+
+```bash
+npm install --save-dev @askui/askui-reporters
+```
+
+## Usage of Allure-Reporter
+Add the reporter to the `UiControlClient` in `jest.setup.ts`:
+
+```typescript
+// Do not forget this import at the start of the file
+import { AskUIAllureStepReporter } from "@askui/askui-reporters";
+...
+  aui = await UiControlClient.build({
+    reporter: new AskUIAllureStepReporter({
+      withScreenshots: 'always' as const, // See below for possible values
+      withDetectedElements: 'always' as const, // See below for possible values
+    })
+  });
+...
+```
+
+You can pass a `ReporterConfig` object to the reporter to configure the level of detail for screenshots and detected elements. The default values are `'onFailure'` for both:
```

Comment:
Can't we link simply over to the corresponding explanation in the askui-reporters package repo so that we don't have to duplicate it here?